css-grid-1/Overview.bs

```diff
@@ -1511,7 +1511,7 @@ Explicit Track Sizing: the 'grid-template-rows' and 'grid-template-columns' prop
 
 		<dfn>&lt;track-size></dfn>          = <<track-breadth>> | minmax( <<inflexible-breadth>> , <<track-breadth>> ) | fit-content( <<length-percentage [0,∞]>> )
 		<dfn>&lt;fixed-size></dfn>          = <<fixed-breadth>> | minmax( <<fixed-breadth>> , <<track-breadth>> ) | minmax( <<inflexible-breadth>> , <<fixed-breadth>> )
-		<dfn>&lt;track-breadth></dfn>       = <<length-percentage [0,∞]>> | <<flex [0,∞]>> | min-content | max-content | auto
+		<dfn>&lt;track-breadth></dfn>       = <<length-percentage [0,∞]>> | <<fraction [0,∞]>> | min-content | max-content | auto
```

Comment:
I think you can remove `[0,∞]` since `<fraction>` is defined to only accept non-negative values, like [`<resolution>`](https://drafts.csswg.org/css-values-4/#resolution) eg. in [`resolution`](https://drafts.csswg.org/mediaqueries-5/#descdef-media-resolution).

Reply:
Done.